vispy/gloo/wrappers.py

```diff
@@ -43,7 +43,7 @@
         depth_test=True,
         cull_face=False,
         blend=True,
-        blend_func=('src_alpha', 'one_minus_src_alpha')),
+        blend_func=('src_alpha', 'one_minus_src_alpha', 'one', 'one_minus_src_alpha')),
```

Comment:
I'm not sure whether we've agreed on this part, as I think @asnt preferred `('src_alpha', 'one_minus_src_alpha', 'zero', 'one')`.

My take is that `.., zero, one` assumes that the destination is initialized with alpha being 1, and that with `.., one,  one_minus_src_alpha` it works "as expected" whatever the destination is initiated with. However, I might be missing something.

Reply:
Ah ok, I saw his comment after I started implementing this and then went to lunch and forgot about it. @asnt what are your thoughts?

Reply:
I think that the destination alpha channel doesn't bring any information we need for the rendering itself. That's why we can discard it when grabbing a screenshot. However, `('src_alpha', 'one_minus_src_alpha', 'zero', 'one')` is convenient because it ensures that the alpha stays in whatever state the user chose it to be initially (most of the time, opaque).

Any other choice of blending functions do not seem to me to bring new capabilities. And this is I think because of step 3 above, in which the initial background 'rgb' colour of the framebuffer always gets blended in, even with a `glClear(0, 0, 0, 0)`, with an `alpha=0`, some black gets mixed with the foreground objects, even if one intended not to have it in.

Reply:
```suggestion
        blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one')),
```

We are keeping in mind that this isn't only for screenshots, right? Are there any non-screenshot cases where this is a bad idea?

Reply:
Probably not. Most common blending functions depend on the source, not the target. And with this, the target's alpha simply stays the same, as @asnt pointed out.